daml-script/runner/src/main/resources/logback.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <logger name="io.netty" level="WARN" />
+    <logger name="io.grpc.netty" level="WARN" />
+
+    <root level="INFO">
+        <appender-ref ref="STDOUT" />
+    </root>
+</configuration>
```

Comment:
I would not put `loback.xml` in the `src/main/resources`. It would get packaged in a jar and gets deployed to maven. This is going to pollute end-user's classpath with unnecessary logback.xml. I would rather force everyone who deploys you app start thinking about logback configuration ASAP :)
`java -Dlogback.configurationFile=/path/to/config.xml` is the way to go. However we have not been following this practice. So feel free to disregard this comment :)

Reply:
`src/test/resources` is fine, it does not pollute production classpath.

Reply:
We don’t plan to publish anything else than the executable and even for the executable you can still override the bundled logback file with `-Dlogback.configurationFile`.